sklearn/metrics/ranking.py

```diff
@@ -250,7 +254,8 @@ def _binary_roc_auc_score(y_true, y_score, sample_weight=None):
             raise ValueError("Only one class present in y_true. ROC AUC score "
                              "is not defined in that case.")
 
-        fpr, tpr, tresholds = roc_curve(y_true, y_score,
+        # use closure for pos_label parameter
```

Comment:
There is a reason why I used closure. `_binary_roc_auc_score ` will be called from `_average_binary_score` as `binary_metric(y_true, y_score, sample_weight=sample_weight)`. And it doesn't look like `_average_binary_score` should know anything about pos_label.